packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -82,7 +85,7 @@ export const ProposalPreview = () => {
         <MainPanel ref={sideNeighborRef}>
           <RowGapBlock gap={24}>
             {/* Statistics */}
-            <Statistics></Statistics>
+            {voteCount && constants && <ProposalStatistics constants={constants} voteCount={voteCount} />}
```

Comment:
I think that we can remove those empty comments now. They just duplicate the code below.

The other thing is that maybe we could show "empty" state as well. I think it will be better to show something while the data is loading rather than not showing anything.

I'd go with a simple `-` (similarly to what we show when loading accounts data for instance).